Monika After Story/game/event-rules.rpy

```diff
@@ -879,11 +877,24 @@ init python:
             IN:
                 per_rules - persistent dict/list to get rules from
             """
+            remove_list = list()

```

Comment:
why are we using a remove list and not just iterating over `per_rules.keys()`?

`.keys` returns a copy of the keys so its fine to use removal with them.